cmd/server/main.go

```diff
@@ -23,6 +24,7 @@ var (
 	defaultProxyTimeoutSeconds = 10
 	defaultRelayUrl            = "https://relay.flashbots.net"
 	defaultRedisUrl            = "localhost:6379"
+	defaultServiceName         = getEnvAsStrOrDefault("Service_Name", "rpc-endpoint")
```

Comment:
env vars should always be capitalized
```suggestion
	defaultServiceName         = getEnvAsStrOrDefault("SERVICE_NAME", "rpc-endpoint")
```